service/src/main/scala/com/yahoo/maha/service/curators/Curator.scala

```diff
@@ -127,8 +127,36 @@ case class DefaultCurator(protected val requestModelValidator: CuratorRequestMod
       )
     } else {
       try {
-        val requestModelResult = requestModelResultTry.get
+        var requestModelResult = requestModelResultTry.get
         requestModelValidator.validate(mahaRequestContext, requestModelResult)
+
+        val isDimOnlyTotalRowQuery: Boolean =
```

Comment:
If we are changing the default curator, to special case a request which we only care about total rows, and the user is already passing in row count of 1, why can't they just pass empty ordering clause ?

Reply:
You're right, the user was passing a problematic query & we shouldn't be updating policy to account for their actions here.  Steps have been taken on that end, and the query is fixed on their end.  Thanks for the input!  Going to close the related PRs for this.